### PR TITLE
offset changes successful

### PR DESCRIPTION
**Pull request by raahi-chada2004**

### CODE REVIEW

File: out_writeapi.go [hunk 1]
```diff
@@ -205,11 +206,29 @@ func sendRequest(ctx context.Context, data [][]byte, config **outputConfig) erro
...
+
+		if (*config).exactlyOnce {
+			appendResult, err = (*config).managedStream.AppendRows(ctx, data, managedwriter.WithOffset((*config).offsetCounter))
+			if err != nil {
+				return err
+			}
+
+			*(*config).appendResults = append(*(*config).appendResults, appendResult)
+
+			err := checkResponses(ms_ctx, (*config).appendResults, true, &(*config).mutex)
```
Comment (agrawal-siddharth) [inline out_writeapi.go:220]: Add a comment here explaining that we will synchronously wait for the response.

File: out_writeapi.go [hunk 2]
```diff
@@ -323,10 +342,13 @@ func FLBPluginInit(plugin unsafe.Pointer) int {
 
 	//set the stream type based on exactly once parameter
 	var currStreamType managedwriter.StreamType
+	var enableRetries bool
```
Comment (agrawal-siddharth) [inline out_writeapi.go:345]: This will be automatically initialized to false, correct? If so, is it necessary to set it to false again below?

**Review by agrawal-siddharth — CHANGES_REQUESTED** [review]

File: out_writeapi.go (hunk 1 above)
Comment (raahi-chada2004) [reply out_writeapi.go:220]: Done.

File: out_writeapi.go (hunk 2 above)
Comment (raahi-chada2004) [reply out_writeapi.go:345]: Done.

**Review by agrawal-siddharth — APPROVED** [review]

**Pull request merged**